.github/workflows/poetry-tests.yaml

```diff
@@ -30,30 +30,88 @@ on:
 
 jobs:
   lint:
-    uses: qutech-delft/github-actions/.github/workflows/_tox-lint.yaml@master
-    with:
-      python-version: ${{ inputs.python-version }}
-      poetry-version: ${{ inputs.poetry-version }}
+    name: Static analysis
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: QuTech-Delft/GitHub-Actions/actions/python/tox-static-check@master
+        with:
+          python-version: ${{ inputs.python-version }}
+          poetry-version: ${{ inputs.poetry-version }}
   unit-test:
-    uses: qutech-delft/github-actions/.github/workflows/_tox-unit-test.yaml@master
-    with:
-      poetry-version: ${{ inputs.poetry-version }}
+    name: Unit testing
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-13 # runs x64
+          - macos-14 # runs arm64
+          - windows-latest
+        python-version:
+          - "3.9"
+          - "3.10"
+          - "3.11"
+          - "3.12"
+    runs-on: ${{ matrix.os }}
+    env:
+      PYTHON_KEYRING_BACKEND: keyring.backends.null.Keyring
```

Comment:
Does this feel a little bit out of place for a high level workflow? Would this be better suited within the tox-unit-test action with a comment about why we need it? (Or if it is not possible to include it in the action just a comment would be fine).

Reply:
Nevermind, out of scope as this was apparently already present before this PR.